### PR TITLE
Mobile view project

### PR DESCRIPTION
**Pull request by Doheera-kosi**

### CODE REVIEW

**Comment by Doheera-kosi** [comment]
Update index.html
Update style.css
Update README.md

File: index.html [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+                <div class="time">04:20 am</div>
+                <div class="status">
+                   <img src="images/status.svg" alt="status" class="status"> 
+                </div>
+            </div>
+            <div class="header-items">
+                <div class="my-logo">
+                    <a class="logo-text">
+                        John Doe
+                    </a>
```
Comment (HAMMAS-SALEEM) [inline index.html:25]: ### Before removing width
before
### After removing width
Screenshot from 2022-08-19 16-52-37


- [ ] Please remove width as shown in the image below to make your logo look like Figma :+1:
**Quick Fix**
Screenshot from 2022-08-19 16-50-49

File: index.html [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+<body>  
+    <div class="screen">
+        <header class="header">
+            <div class="time-status">
+                <div class="time">04:20 am</div>
+                <div class="status">
+                   <img src="images/status.svg" alt="status" class="status"> 
+                </div>
+            </div>
+            <div class="header-items">
```
Comment (HAMMAS-SALEEM) [inline index.html:21]: white-space (copy)


- [ ] Please add the following code at the very start of your style.css file to remove the white space on top and left :+1:

```
body {
  margin: 0;
  padding: 0;
  box-sizing: border-box;
}
```

**Review by HAMMAS-SALEEM — CHANGES_REQUESTED** [review]
# Note: You have created a separate pull request for a re-reivew of your project. Please don't do this, always use the same pull request for each project and don't make a new repository or pull request for re-review of same milestone
 
# Required Changes :recycle:
almost-there

### Hey @Doheera-kosi :wave:

### Highlights :1st_place_medal:
 
:heavy_check_mark: Meaningful commits
:heavy_check_mark: Added .gitignore
:heavy_check_mark: Following GitHub Flow

Good job on implementing some of the changes requested by the previous reviewer, there are some issues that you still need to work on to go to the next project but you are almost there! :blush:

# Optional Suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @HAMMAS-SALEEM in your question so I can receive the notification. Otherwise reach out to me on slack which is more fast. You can reach out to me on slack by searching my name in **full-time-students** channel :+1:

_**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_

------------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request closed without merging**